lib/middleware.ts

```diff
@@ -113,6 +113,97 @@ var defaultHandlers = [
   () => ({title: "request-finished"}),
 ];
 
+const defaultContextHandlers = [
+  (req) => ({"request-id": req.headers["x-request-id"]}),
+  (req) => ({"session-id": req.sessionID}),
```

Comment:
The kayvee middleware doesn't add `x-request-id`. It's something that must be set independently.
Personally, I think it's useful to add `request-id` and `session-id` as defaults even if they might not always exist but open to feedback here.